### PR TITLE
Changed extends 'base.html' to extends 'generic/object_detail.html' for detail templates

### PR DESCRIPTION
**Pull request by briddo**
Fixes: #33

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (glennmatthews) [inline .idea/.gitignore:1]: Looks like we need a blanket ignore for `.idea` in the project's base `.gitignore` - these files shouldn't be added.

File: nautobot_data_validation_engine/templates/nautobot_data_validation_engine/minmaxvalidationrule.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{% extends 'base.html' %}
+{% extends 'generic/object_detail.html' %}
```
Comment (glennmatthews) [inline nautobot_data_validation_engine/templates/nautobot_data_validation_engine/minmaxvalidationrule.html:1]: `object_detail.html` was only added in Nautobot 1.2, so making this change will also need to raise the minimum required Nautobot version for this plugin correspondingly (`min_version` in `nautobot_data_validation_engine/__init__.py` as well as documentation).

File: nautobot_data_validation_engine/templates/nautobot_data_validation_engine/regularexpressionvalidationrule.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
-{% extends 'base.html' %}
+{% extends 'generic/object_detail.html' %}
 {% load buttons %}
 {% load static %}
-{% load helpers %}
+{% load helpers %}s
```
Comment (glennmatthews) [inline nautobot_data_validation_engine/templates/nautobot_data_validation_engine/regularexpressionvalidationrule.html:4]:
```suggestion
{% load helpers %}
```

**Review by glennmatthews — CHANGES_REQUESTED** [review]

File: .idea/.gitignore (hunk 1 above)
Comment (briddo) [reply .idea/.gitignore:1]: Yeah sorry about that! I'm too used to doing `git add .` and not even checking what's been added.

File: nautobot_data_validation_engine/templates/nautobot_data_validation_engine/minmaxvalidationrule.html (hunk 2 above)
Comment (glennmatthews) [inline nautobot_data_validation_engine/templates/nautobot_data_validation_engine/minmaxvalidationrule.html:1]: In addition to this change, need to review whether any or all of the contents of `block title` and `block header` are now redundant, and if they can be removed to simplify the template.

Additionally, `block content` should be updated to only override `block content_left_page` as appropriate.

Same comment on the other template file below.

File: nautobot_data_validation_engine/__init__.py [hunk 4]
```diff
@@ -23,6 +23,7 @@ class NautobotDataValidationEngineConfig(PluginConfig):
     required_settings = []
     default_settings = {}
     caching_config = {}
+    min_version = "1.2"
```
Comment (glennmatthews) [inline nautobot_data_validation_engine/__init__.py:26]: Would be good to also add `nautobot ~1.2` in `pyproject.toml` as well (currently missing), if that doesn't open too much of a can of worms to do so at this time...

**Pull request closed without merging**